src/commands/dev.ts

```diff
@@ -371,6 +371,8 @@ export async function command(commandOptions: CommandOptions) {
           const newPath = reqPath.substr(workerConfig.args.toUrl.length);
           try {
             const response = await got(`${workerConfig.args.fromUrl}${newPath}`, {
+              // @ts-ignore - TS doesn't like method being a generic string
+              method: req.method,
```

Comment:
I'm not sure of the exact TS fu that's needed here?

Reply:
I'll fix and then merge